functions/src/definitions/eventHandlers/onInstanceCreate.ts

```diff
@@ -21,9 +22,9 @@ if (!admin.apps.length) {
   admin.initializeApp()
 }
 
-const onInstanceCreate = functions
-  .region("asia-southeast1")
-  .runWith({
+const onInstanceCreateV2 = onDocumentCreated(
+  {
+    document: "messages/{messageId}/instances/{instanceId}",
```

Comment:
```suggestion
    document: "/messages/{messageId}/instances/{instanceId}",
```

Reply:
The examples in the documentation seem to go without the leading slash so I went with it!